### DISCUSSION TITLE
Does COD/warzone work well when ZAPRET is turned on?

### DISCUSSION THREAD

**Discussion by ilyakozban** [bol-van/zapret#755] [Q&A] [answered]
Hi. Im facing a problem in call of duty when https://github.com/Flowseal/zapret-discord-youtube is turned on. The game crashing immediately. And the solution that i found is only to remove services before launching. 
So please, can someone let me know if zapret works well with COD or if facing with such a problem and know how fix it i would be very grateful.

**Comment by bol-van** [comment]
zapret is not flowseal. zapret is a complex software that requires fine tuning for your ISP.
flowseal is zapret+predefined config.  this config can break things.
tuning is not easy and requires skills. If you can tune it you probably can avoid your problem.
If you expect one-button solution the answer for you is NO

**Reply by ilyakozban** [reply]
Спасибо за ответ. Я прекрасно понимаю что zapret намного комплексей и сложнее в реализации. К сожалению не обладаю достаточными навыками и знаниями чтобы самостоятельно решить проблемы в вылетами, поэтому и спрашиваю. Возможно кто-то сталкнулся с подобным или наоборот, у человека нет такой проблемы с использованиям запрета и в таком случае это сильно бы мне помогло.

**Comment by numbereleven-a** [comment, marked as answer]
> Привет. Я столкнулся с проблемой в Call of Duty, когда включен https://github.com/Flowseal/zapret-discord-youtube . Игра сразу же вылетает. И решение, которое я нашел, это только удаление служб перед запуском. Так что, пожалуйста, может ли кто-нибудь сообщить мне, хорошо ли работает Zapret с COD или если вы столкнулись с такой проблемой и знаете, как ее исправить, я был бы очень признателен.

На это многие жаловались, некоторым системам античита не нравится windivert (или как то так он называется), я бы в целом не рекомендовал играть в онлайн игры с включённым запретом дабы не попасть под случайный банхамер, можно написать простой скрипт на павершеле который будет вам проверять наличие запущенной службы, убивать её и запускать игру.

**Comment by TurboKirill** [comment]
Да, работает с включенным zapret, он никак не отражается. Но проблема была у самого COD.
Я запускал игру с GamePass, сначала вылетала и не хотела включаться, отправил репорт, пришло какое-то обновление, перезагрузил ПК и игра запустилась. В дальнейшем вылетов не было, если не считать, что она выключается при запуске компании и запускается вновь сразу в компанию.
